shared_libraries/orca_shared/database/use_cases/validation.py

```diff
@@ -27,6 +26,18 @@ def _validate_password(password: str, context: str, logger: logging.Logger) -> N
         msg = f"{context} password must be at least 12 characters long."
         logger.critical(msg)
         raise Exception(msg)
+    elif re.search('[0-9]', password) is None:
+        msg = f"{context} password must contain a digit between 0 and 9."
+        logger.critical(msg)
+        raise Exception(msg)
+    elif re.search('[A-Z]', password) is None:
```

Comment:
I'm tempted to say we should require lower and upper case.

Reply:
yup added.